contracts/stake/src/multitest/suite.rs

```diff
@@ -1,27 +1,29 @@
-use std::collections::HashMap;
+use std::{collections::HashMap, ops::Add};
```

Comment:
What is this for?

Reply:
I might have added it by accident, or it is there from before. Removing it